src/pages/SetPasswordPage.js

```diff
@@ -38,8 +38,12 @@ const propTypes = {
         password: PropTypes.string,
     }),
 
-    // The accountID and validateCode are passed via the URL
-    route: validateLinkPropTypes,
```

Comment:
Looks like maybe these should not have been removed.

Reply:
ah validateLinkPropTypes seemed confusing to me, but I guess it makes sense since this set password endpoint also handles validating accounts.